config/config_helpers_test.go

```diff
@@ -492,6 +493,27 @@ TERRAGRUNT_HIT","")}/bar`,
 			"foo/HIT/bar",
 			nil,
 		},
+		{
+			`"${get_terraform_commands_that_need_locking()}"`,
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/" + DefaultTerragruntConfigPath, NonInteractive: true},
+			util.CommaSeparatedStrings(TERRAFORM_COMMANDS_NEED_LOCKING),
+			nil,
+		},
+		{
+			`commands = ["${get_terraform_commands_that_need_vars()}"]`,
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/" + DefaultTerragruntConfigPath, NonInteractive: true},
+			fmt.Sprintf("commands = [%s]", util.CommaSeparatedStrings(TERRAFORM_COMMANDS_NEED_VARS)),
+			nil,
+		},
+		{
+			`commands = "test-${get_terraform_commands_that_need_vars()}"`,
+			nil,
+			options.TerragruntOptions{TerragruntConfigPath: "/root/child/" + DefaultTerragruntConfigPath, NonInteractive: true},
+			fmt.Sprintf(`commands = "test-%v"`, TERRAFORM_COMMANDS_NEED_VARS),
+			nil,
+		},
```

Comment:
Nice tests! Could you add one more that calls two interpolation functions in one string?